src/storage.cc

```diff
@@ -171,7 +181,12 @@ Status Storage::Open(bool read_only) {
   }
 
   rocksdb::BlockBasedTableOptions metadata_table_opts;
-  metadata_table_opts.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10, true));
+  metadata_table_opts.format_version = 5;
```

Comment:
Although rocksdb will do some optimization if format_version is 5, i still want to set those options explicitly.

Otherwise, i think we should enable `pin_l0_filter_and_index_blocks_in_cache`, WDYT @git-hulk

Reply:
I also agree that should enable `pin_l0_filter_and_index_blocks_in_cache` by default which would bring better performance with a little pin memory cost.